advertise.md

```diff
@@ -0,0 +1,86 @@
+          <label class="_hover-text">
+            JDBC interpreter lets you create a JDBC connection to any data sources seamlessly. Postgresql, Mysql, MariaDB, Redshift, Apache Hive and so on.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/sql_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top:65px;">
+            <img src="./assets/themes/zeppelin/img/python-logo-generic.svg" width="200px">
+          </label>
+          <label class="_hover-text">
+            Python is supported with Zeppelin. Also integrated Matplotlib with Python & Pyspark interpreter, and Conda is now available in Zeppelin.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/python_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-12 col-sm-12 col-xs-12 text-center">
+      <p style="text-align:center; margin-top: 20px; font-size: 17px; color: gray; font-weight: 200; font-style: italic; padding-bottom: 0;">
+        See more details in Zeppelin supports 20+ different interpreters.
+        <a href="/docs/0.8.0-SNAPSHOT/manual/interpreters.html">LEARN MORE <span class="glyphicon glyphicon-chevron-right" style="font-size:15px;"></span></a>
+      </p>
+    </div>    
+    <hr />
+    <div class="border row">
+      <h2>Deployments</h2>
+      <div class="border col-md-6 col-sm-6">
+        <p class="panel-content-user">
+          <label style="width: 100%;">
+            <span class="glyphicon glyphicon-user" style="font-size:30px;position:relative;width:100%;text-align:center;"></span>
+            <span class="title-text">Single User</span>            
+          </label>
+          <label class="content-text">
+            Local Spark, 6 Built-in visualizations, Display system, Dynamic form, Multiple backends are supported.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/tutorial.html" class="user-button">LEARN MORE</a>
+          </label>
+        </p>
+      </div>
+      <div class="border col-md-6 col-sm-6">
+        <p class="panel-content-user">
+          <label style="width: 100%;">
+            <span class="fa fa-users" style="font-size:30px;position:relative;width:100%;text-align:center;"></span>
+            <span class="title-text">Multi-User</span>
+          </label>
+          <label class="content-text">
+            Zeppelin supports Multi-user Support w/ LDAP. Let's configure Zeppelin for your yarn cluster.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/usage/other_features/personalized_mode.html" class="user-button" style="margin-top:15px">LEARN MORE</a>
```

Comment:
1. Need more `margin-top`.  25px would be proper. 
2. would be nice to add `padding-bottom: 15px` for `.advertise .panel-content-user` (decreasing 10px) since we add `10px` to margin-top.



![image](https://cloud.githubusercontent.com/assets/4968473/26774914/7a9daed0-4a0d-11e7-972a-be1843eab5e5.png)